docs/eventing/sources/rabbitmq-source/README.md

```diff
@@ -0,0 +1,219 @@
+# Creating a RabbitMQ Source
+
+This topic describes how to create a RabbitMQ Source.
+
+## Prerequisites
+
+To use the RabbitMQ Source, you must have the following installed:
+
+1. [Knative Eventing](../../../install/yaml-install/eventing/install-eventing-with-yaml.md)
+1. [RabbitMQ Cluster Operator (optional)](https://github.com/rabbitmq/cluster-operator) - our recommendation is [latest release](https://github.com/rabbitmq/cluster-operator/releases/latest)
+1. [CertManager v1.5.4](https://github.com/jetstack/cert-manager/releases/tag/v1.5.4) - easiest integration with RabbitMQ Messaging Topology Operator
+1. [RabbitMQ Messaging Topology Operator](https://github.com/rabbitmq/messaging-topology-operator) - our recommendation is [latest release](https://github.com/rabbitmq/messaging-topology-operator/releases/latest) with CertManager
+
+## Install the RabbitMQ controller
+
+1. Install the RabbitMQ Source controller by running the command:
+
+    ```bash
+    kubectl apply -f {{ artifact(org="knative-sandbox", repo="eventing-rabbitmq", file="rabbitmq-source.yaml") }}
+    ```
+
+1. Verify that `rabbitmq-controller-manager` and `rabbitmq-webhook` are running:
+
+    ```bash
+    kubectl get deployments.apps -n knative-sources
+    ```
+
+    Example output:
+
+    ```{ .bash .no-copy }
+    NAME                           READY   UP-TO-DATE   AVAILABLE   AGE
+    rabbitmq-controller-manager     1/1     1            1           3s
+    rabbitmq-webhook                1/1     1            1           4s
+    ```
+
+## Create a RabbitMQ cluster (optional)
```

Comment:
does this need to be in our docs at all, or could it just be a link in the prereqs to rabbitmq docs that describe how to create a cluster?

Reply:
true, removing it! Ty :)